eclipse-collections-api/src/main/java/org/eclipse/collections/api/map/MutableMapIterable.java

```diff
@@ -161,11 +161,15 @@ default boolean removeIf(Predicate2<? super K, ? super V> predicate)
      * map = map.withMap(someMap);
      * </pre>
      *
-     * @see #put(Object, Object)
+     * @see #putAll(Map)
      *
      * @since 10.3.0
      */
-    MutableMapIterable<K, V> withMap(Map<K, V> map);
+    default MutableMapIterable<K, V> withMap(Map<K, V> map)
```

Comment:
While I do think we should pull the method up, we need it to have the correct generics, which is a breaking change. It needs the bounded wildcards ? extends K, ? extends V

Reply:
Oh yes! That is true. Fixing it right away